src/Sylius/Bundle/PromotionBundle/Form/Type/Action/PercentageDiscountConfigurationType.php

```diff
@@ -36,6 +37,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     new Type(['type' => 'numeric']),
                 ],
             ])
+            ->add('filters', ActionFiltersType::class)
```

Comment:
it is declared as service here: https://github.com/Sylius/Sylius/pull/6054/files#diff-17b78b75b647ab8c629e081f3ee1af1fR122